text/0035-opentelemetry-protocol.md

```diff
@@ -0,0 +1,425 @@
+# OpenTelemetry Protocol Specification
+
+_Author: Tigran Najaryan, Omnition Inc._
```

Comment:
Usually we don't use _Author.

Reply:
It's not a bad idea to recognize authors and contributors this way (e.g. @akehlenbeck whose comments changed the OTEP in a significant way from the first version)

Reply:
I am fine either way, please feel free to make an "executive TC decision" and I will apply it :-)